src/vs/platform/native/electron-main/nativeHostMainService.ts

```diff
@@ -161,7 +161,8 @@ export class NativeHostMainService extends Disposable implements INativeHostMain
 				addMode: options.addMode,
 				gotoLineMode: options.gotoLineMode,
 				noRecentEntry: options.noRecentEntry,
-				waitMarkerFileURI: options.waitMarkerFileURI
+				waitMarkerFileURI: options.waitMarkerFileURI,
+				remoteAuthority: options.remoteAuthority || undefined
```

Comment:
Maybe rather `withNullAsUndefined(options.remoteAuthority)`? Does the `null` not travel to the main side?